app/views/doorkeeper/authorizations/new.html.erb

```diff
@@ -13,7 +13,7 @@
 <h3>Authorization Required</h3>
 
 <p class="h4">
-  <%= t('.prompt', client_name: "<strong class=\"text-info\">#{ @pre_auth.client.name }</strong>") %>
+  <%= raw t('.prompt', client_name: "<strong class=\"text-info\">#{ h @pre_auth.client.name }</strong>") %>
```

Comment:
See #1520 for context. The PR removed the `raw` tag because of the potential XSS via the client name. However, this escapes the `<strong> ... </strong>` tags too. Now, restore the `raw` tag but escape only the client name.

Reply:
Is the h before the `@pre_auth.client.name` for here how to specify that it should be escaped?

Reply:
Yes, see https://api.rubyonrails.org/classes/ERB/Util.html#method-c-h